script/oracle/install-instantclient-packages.sh

```diff
@@ -42,6 +42,6 @@ fi
 
 # hack for system-builder ENV
 rm -rf /opt/system/vendor/oracle
-ln -sf /opt/oracle/instantclient_19_6/libsqora.so.19.1 /opt/oracle/instantclient_19_6/libsqora.so
-ln -sf /opt/oracle/instantclient_19_6 /opt/oracle/instantclient
+ln -sf /opt/oracle/instantclient_19_18/libsqora.so.19.1 /opt/oracle/instantclient_19_18/libsqora.so
```

Comment:
why this particular version?

Reply:
Because `instantclient_19_6` doesn't exist in the Oracle repo anymore. This 19_18 is the latest patch version and the one they currently provide.